build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
My custom mode-line is here: https://github.com/stardiviner/emacs.d/blob/master/init/Emacs/init-my-emacs-mode-line.el#L537.

And I use custom `build-status` function in my custom mode-line to replace `build-status-mode-line-string`.
```elisp
(defun *build-status ()
  "Show CI build status in mode-line."
  ;; `build-status-mode-line-string'
  (let* ((root (or (build-status--circle-ci-project-root (buffer-file-name))
                   (build-status--travis-ci-project-root (buffer-file-name))))
         (status (cdr (assoc root build-status--project-status-alist)))
         (project (build-status--project (buffer-file-name))))
    (if (and (build-status--activate-mode) (null status))
        (propertize
         (concat
          (all-the-icons-faicon "cogs"
                                :v-adjust -0.05)
          (propertize " " 'face 'variable-pitch)
          (cond
           ((string= status "passed")
            (all-the-icons-faicon "check-circle")
            (build-status--propertize "P" status))
           ((string= status "running")
            (all-the-icons-faicon "spinner")
            (build-status--propertize "R" status))
           ((string= status "failed")
            (all-the-icons-faicon "chain-broken")
            (build-status--propertize "F" status))
           ((string= status "queued")
            (all-the-icons-faicon "ellipsis-h")
            (build-status--propertize "Q" status))
           (t
            (all-the-icons-faicon "question-circle-o")
            (build-status--propertize
             "?"
             (replace-regexp-in-string "[^a-zA-Z0-9[:space:]]+" " "
                                       (or status "unknown")))))
          )))))
```